package-lock.json

```diff
@@ -3693,7 +3693,7 @@
       "dependencies": {
         "domelementtype": {
           "version": "1.1.3",
-          "resolved": "https://registry.npmjs.org/domelementtype/-/domelementtype-1.1.3.tgz",
```

Comment:
uncommit this file

Reply:
not exactly sure how to uncommit this correctly